internal/config/config.go

```diff
@@ -61,9 +61,10 @@ type PodInfo struct {
 }
 
 func (c TLSConfig) CertificatesConfigured() bool {
```

Comment:
Is this func still needed? It looks like all calls/references are being removed.

Reply:
Good spot, thanks!